src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerTest {
+    private OperationHandler purchaseHandler;
+    private FruitTransaction transaction;
+
+    @BeforeEach
+    public void setUp() {
+        purchaseHandler = new PurchaseHandler();
+        transaction = new FruitTransaction(FruitTransaction.Operation.PURCHASE,"apple", 1);
```

Comment:
Move this lines to the method beforeAll(), like I suggested in previous comments.
Fix all similar cases in all test classes where needed.